INSTALL.md

```diff
@@ -164,10 +164,28 @@ Some of our examples will also need **libSplash**.
       [LD\_LIBRARY\_PATH](#additional-required-environment-variables-for-optional-libraries)
       to `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ADIOS_ROOT/lib`
 
-- **ISAAC** (requires *boost mpl*, *boost fusion*, *IceT*, *Jansson*, *libjpeg*)
-    - Enables live in situ visualization, see more
-      - [General installation informations](https://github.com/ComputationalRadiationPhysics/isaac/blob/master/INSTALL.md)
-      - [PIConGPU specific installation informations](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Plugin%3A-ISAAC)
+- **ISAAC** (requires *boost* (header only), *IceT*, *Jansson*, *libjpeg*)
```

Comment:
you also need `libwebsockets` and to be more precise, let us suggest `libjpeg-turbo` to be useful :)